.github/workflows/pack_cwls.yml

```diff
@@ -0,0 +1,39 @@
+name: Pack CWL Files
+
+# Triggers the action on push or pull requests,
+# but only for the develop branch
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  pack_cwls:
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.5, 3.6, 3.7]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install cwltool
+        run: |
+          python -m pip install toil-msk[cwl]
+      - name: Run cwltool --pack for all workflows
+        run: |
+          echo Removing old packed workflows
+          rm -rf ./packed_workflows/*
+          echo Packing CWL files...
+          find . -name '*.cwl' ! -path '*/command_line_tools/*' ! -path '*/packed_workflows/*' -type f -print0 | xargs -0 -I file sh -c 'base=$(basename file); cwltool --pack file > packed_workflows/${base/.cwl/__packed.cwl}'
```

Comment:
Looks good, what happens when it fails. Also, should we do `cwltool --validate __packed.cwl`

Reply:
The check will fail and the packed file for that workflow will not be created:

<img width="1680" alt="Screen Shot 2020-09-10 at 11 55 27 AM" src="https://user-images.githubusercontent.com/6385646/92758393-ab9dae80-f35c-11ea-98cc-01f2a7363502.png">

I've also added a validate step afterwards